### PR TITLE
refactoring into smaller piecies

### PR DESCRIPTION
**Pull request by deep-diver**
decouple files under `model` directory

- `common.py` to define common variables like key names (i.e. `IMAGE_KEY`)
- `utils.py` to define common utility function like log and `transformed_key`
- `hyperparams.py` to define a set of default hyperparameters
- `signatures.py` to define the signature for the `serving_default`.

### CODE REVIEW

File: training_pipeline/models/signatures.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+def _serving_preprocess(string_input):
+    """
+    _serving_preprocess turns base64 encoded string data into a
+    string type of Tensor. Then it is decoded as 3 channel based
+    uint8 type of Tensor. Finally, it is normalized and resized
+    to the size the model expects.
+    """
+    decoded_input = tf.io.decode_base64(string_input)
+    decoded = tf.io.decode_jpeg(decoded_input, channels=3)
+    decoded = decoded / 255
```
Comment (sayakpaul) [inline training_pipeline/models/signatures.py:20]: Let's use `mobilenet_v2.preprocess_input()` for consistency.

File: training_pipeline/models/signatures.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        into the model. The model outputs predictions of the request p
+        ayloads, then it will be returned back to the client after app
+        lying postprocess of tf.math.argmax to the outputs(logits)
+        """
+        images = _serving_preprocess_fn(string_input)
+        logits = m_call(**images)
+        seg_mask = tf.math.argmax(logits, -1)
+        return {"seg_mask": seg_mask}
+
+    return serving_fn
```
Comment (sayakpaul) [inline training_pipeline/models/signatures.py:67]: Nice 🔥

**Review by sayakpaul — CHANGES_REQUESTED** [review]
Nice work! Could you add a README or brief descriptions at the top of each of the individual scripts to denote what they're responsible for doing?

**Comment by deep-diver** [comment]
> Nice work! Could you add a README or brief descriptions at the top of each of the individual scripts to denote what they're responsible for doing?

Absolutely!

**Comment by deep-diver** [comment]
@sayakpaul 

addressed your comment :)

**Review by sayakpaul — APPROVED** [review]
Excellent update.

**Pull request merged**